src/Models/CsvCatalogItem.js

```diff
@@ -239,14 +301,14 @@ CsvCatalogItem.prototype._showInCesium = function() {
             return when(featurePromise, function(results) {
                 if (defined(results)) {
```

Comment:
Actually, wouldn't a loop be better?  Currently NM only shows the first picked feature, so updating the others is arguably unnecessary, but this is likely to change soon.  See #569.